### DISCUSSION TITLE
Cannot style image background color in README

### DISCUSSION THREAD

**Discussion by AntoniosBarotsis** [community/community#41899] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I'm not sure if this is a bug or not but I can't seem to see the background color of an `img` tag (or the underlying `svg`) and have that change reflected in a README file.

I have the following tag

```html
<p align="center">
  <img src="assets/plot.svg" alt="Benchmark" width="600" style="background-color:#2e3136">
</p>
```

Which makes the image look like the following in VS Code
![image](https://user-images.githubusercontent.com/50240570/208063973-22fc0e63-8144-4473-bd14-a68ebf0ea7ff.png)

However, the [README](https://github.com/AntoniosBarotsis/qr-rs) remains the same regardless of whether I add the style or not. 
![image](https://user-images.githubusercontent.com/50240570/208065624-e5a7bfd1-e3a0-4358-98d2-6ac4e57e8b55.png)

As I mentioned earlier, I tried adding the background color in the `svg` file itself, which again rendered correctly in VS Code but not on Github.

**Comment by samharp** [comment, marked as answer]
Sadly, GitHub strips custom CSS from Readme files when rendering on the site ( https://css-tricks.com/custom-styles-in-github-readmes/ ). To get around this, you could try adding a large rectangle in SVG itself (creating a shape in the background)?

**Comment by mohamedzhioua** [comment]
here ( https://stackoverflow.com/questions/14494747/how-to-add-images-to-readme-md-on-github )
There is this [highly cited question](https://stackoverflow.com/questions/14494747/add-images-to-readme-md-on-github) with many answers.

Those answers are about including pictures in a markdown file, as rendered by GitHub in [GFM (GitHub Flavored Markdown)](https://github.github.com/gfm/#images)

There is no support for background image in that case.

Only [GitHub Pages](https://pages.github.com/) would be able to render those files as HTML, with possibly background images, as part of a static website generator.
